fehm_toolkit/heat_in.py

```diff
@@ -13,10 +13,6 @@
 
 logger = logging.getLogger(__name__)
 
-HEATFLUX_INPUT_ZONE = 'bottom'
-HEATFLUX_HEADER = 'hflx\n'
-HEATFLUX_FOOTER = '0\n'
-
```

Comment:
I realised after chatting with Kristin about this that these variables are only used in one place - I think it's clearer just putting them in-line directly.

Reply:
Aha, good point. Perhaps they will be used elsewhere at a later time.